android/src/main/java/nl/fhict/happynews/android/activity/SourcesSettingsActivity.java

```diff
@@ -1,23 +1,27 @@
 package nl.fhict.happynews.android.activity;
 
+import android.content.Intent;
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
 import android.widget.ListView;
 import com.google.gson.Gson;
 import nl.fhict.happynews.android.R;
-import nl.fhict.happynews.android.SourceManager;
```

Comment:
It looks like the `nl.fhict.happynews.android.SourceManager;` import is required.